test/core/network/types/bootstrap_nodes_test.cpp

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "network/types/bootstrap_nodes.hpp"
+
+#include <gmock/gmock.h>
+
+#include <libp2p/peer/peer_info.hpp>
+
+#include "mock/core/application/app_configuration_mock.hpp"
+#include "mock/core/application/chain_spec_mock.hpp"
+#include "testutil/outcome.hpp"
+
+using namespace kagome::application;
+using namespace kagome::network;
+using namespace libp2p;
+using ::testing::ReturnRef;
+
+peer::PeerId operator""_peerid(const char *c, size_t s) {
```

Comment:
Maybe put it in literals.hpp?

Reply:
Hm. It already exists there. Will be used literal from testutils